task_2/style.css

```
@@ -0,0 +1,60 @@
+.wrapper {
+    margin: 0 auto;
+    width: 700px;
+}
+
+body {
+    background-color: rgb(231, 238, 238);
+}
+
+.header {
+    height: 110px;
+    padding-top: 10px;
+    text-align: center;
+    width: 100%;
+}
+
+.header-title {
+    font-size: 25px;
+    margin-bottom: 20px;
+    margin-right: 15px;
+}
+
+.main {
+    height: 790px;
+}
+
+form {
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+    margin-top: 20px;
+}
+
+::-webkit-input-placeholder {
```

You have to handle it for more browser 
```suggestion
::placeholder { /* Chrome, Firefox, Opera, Safari 10.1+ */
 ```